CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   * `cortex_alertmanager_silences`
 * [CHANGE] Distributor: Drop experimental `-distributor.direct-otlp-translation-enabled` flag, since direct OTLP translation is well tested at this point. #9647
 * [CHANGE] Ingester: Change `-initial-delay` for circuit breakers to begin when the first request is received, rather than at breaker activation. #9842
+* [CHANGE] Query-frontend: run the pruning middleware before the query sharding middleware instead of after. #9913
```

Comment:
[nit] Probably makes sense to describe this in terms the average user would understand:
```suggestion
* [CHANGE] Query-frontend: apply query pruning before query sharding instead of after. #9913
```